src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -11,12 +11,183 @@
         <fullName>Always_Use_Last_Day_Of_Month__c</fullName>
         <defaultValue>false</defaultValue>
         <externalId>false</externalId>
-        <inlineHelpText>When checked, all future monthly or quarterly donation installments will be set to the last day of the month, regardless of the actual date in the &quot;Next Donation Date&quot; field. This is ignored when the installment period is not set to Monthly or Quarterly.</inlineHelpText>
+        <inlineHelpText>Sets future monthly, custom monthly, or quarterly donation installments to the last day of the month, regardless of Next Donation Date. Installment period must be Monthly or Quarterly, or a custom installment period using months.</inlineHelpText>
```

Comment:
@jjbennett Assuming this checkbox overrides the DayOfMonth field value (going by the above unit test), should that be noted in the InlineHelpText (if there's room)?

Reply:
Yes Always always wins, @sam-knox thoughts on whether or not to include something in the help text and if so, what?

Reply:
There's already some validation code in the `RD_RecurringDonations_TDTM` class (line 97) around the AlwaysUseLastDayOfMonth field. Do you think it's worth adding logic there to advise the User to not use the DayInMonth field when the AlwaysUseLastDayOfMonth field is checked?

Reply:
@force2b - It doesn't hurt anything to have then both set, I think if we explain the hierarchy in the docs we shouldn't need to prevent the user from saving with a value in both fields. @mpusto thoughts?